File: src/main/java/com/sfh/pokeRogueBot/model/browser/pokemonjson/SummonData.java
package com.sfh.pokeRogueBot.model.browser.pokemonjson;

import lombok.Data;

import java.util.List;

@Data
public class SummonData {

    private List<Integer> battleStats;
    private List<Object> moveQueue;
    private int disabledMove;
    private int disabledTurns;
    private List<Object> tags;
    private boolean abilitySuppressed;
    private int ability;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/browser/pokemonjson/SummonData.kt
package com.sfh.pokeRogueBot.model.browser.pokemonjson

data class SummonData(
    val battleStats: List<Int>?,
    val moveQueue: List<Any>?,
    val disabledMove: Int,
    val disabledTurns: Int,
    val tags: List<Any>?,
    val abilitySuppressed: Boolean,
    val ability: Int
)